### PR TITLE
bug fixes to GDrive crawler

### PR DESCRIPTION
**Pull request by ofermend**

### CODE REVIEW

File: crawlers/CRAWLERS.md [hunk 1]
```diff
@@ -328,6 +328,31 @@ The HubSpot crawler has no specific parameters, except the `HUBSPOT_API_KEY` tha
...
+      - jana@vectara.com
+```
+
+The gdrive crawler indexes content of your Google Drive folder
+- `days_back`: include only files created within the last N days
+- `permissions`: list of `displayName` values to include. We recommend including your company name (e.g. `Vectara`) and `all` to include all non-restricted files.
+- `delegated_users`: list of user emails in your organization. 
+- `ray_workers`: 0 if not using Ray, otherwise specifies the number of Ray workers to use.
+
+This crawler identified Google Drive files based on the list of delegated users. For each user it looks at those files that the user either created or has access to, but
```
Comment (amin2718) [inline crawlers/CRAWLERS.md:350]: This crawler identifies

**Review by amin2718 — APPROVED** [review]
Hi Ofer,

Thanks! As I reviewed the PR, I realized that you've made many substantial improvements to the original crawler. I appreciate you moving this towards being more "production ready".

File: crawlers/CRAWLERS.md [hunk 2]
```diff
@@ -328,6 +328,32 @@ The HubSpot crawler has no specific parameters, except the `HUBSPOT_API_KEY` tha
...
+- `days_back`: include only files created within the last N days
+- `permissions`: list of `displayName` values to include. We recommend including your company name (e.g. `Vectara`) and `all` to include all non-restricted files.
+- `delegated_users`: list of user emails in your organization. 
+- `ray_workers`: 0 if not using Ray, otherwise specifies the number of Ray workers to use.
+
+This crawler identifies Google Drive files based on the list of delegated users. For each user it looks at those files that the user either created or has access to, but
+limiting only to files that have "accessible by all" permissions (so that "restricted" files are not included)
+
+Note that this crawler uses a Google Drive service account mode to access files, 
+and you need to include a `credentials.json` file in the main vectara-ingest folder.
```
Comment (adeelehsan) [inline crawlers/CRAWLERS.md:354]: May be we can add the information about the content of the `credentials.json` ?

Comment (ofermend) [reply crawlers/CRAWLERS.md:354]: Good idea. Can you please suggest some text or a link to Google API docs that describe it?

**Pull request merged**